CMakeLists.txt

```diff
@@ -1,6 +1,105 @@
-# CMakeLists.txt has to be located in the project folder and cmake has to be
-# executed from 'project/build' with 'cmake ../'.
-cmake_minimum_required(VERSION 2.6)
-find_package(Rock)
-rock_init(vizkit3d_normal_depth_map 0.1)
-rock_standard_layout()
+cmake_minimum_required(VERSION 2.8.3)
+project(gazebo_ros_sonar_plugin)
```

Comment:
Why are the repository name and project name different? Some tools assume they are always the same...

Reply:
Ok